src/v/config/node_config.cc

```diff
@@ -149,6 +149,13 @@ node_config::node_config() noexcept
       "than the cluster's consensus version",
       {.visibility = visibility::tunable},
       false)
+  , memory_allocation_warning_threshold(
+      *this,
+      "memory_allocation_warning_threshold",
+      "Enables log messages for allocations greater than or equal to the given "
```

Comment:
nit: Shouldn't this be "greater than given size." Per code..

```
 761 void
 762 inline
 763 cpu_pages::check_large_allocation(size_t size) {
 764     if (size > large_allocation_warning_threshold) {
 765         warn_large_allocation(size);
 766     }
 767 }
 768 
```

or am I looking in the wrong place?

Reply:
oh nice catch

Reply:
Nice! Must've missed remembered things. Changing the description now.